source/code-snippets/usage-examples/replaceOne.js

```diff
@@ -14,35 +14,14 @@ async function run() {
     const movies = database.collection("movies");
 
     // create a query for a movie to update
-    const query = { title: "Blacksmith Scene" };
-    const options = {
-      // create a document if no documents match the query
-      upsert: true,
-    };
+    const query = { title: { $regex: "The Cat from" } };
     // create a new document that will be used to replace the existing document
     const replacement = {
-      title: "Sandcastles in the Sand",
-      plot:
-        "Robin Sparkles mourns for a relationship with a mall rat at an idyllic beach.",
+      title: `The Cat from Sector ${Math.random()}`
```

Comment:
Super minor, but this will be some random number between 0 and 1 with many decimal places. Might be nicer to bound it like `Math.floor(Math.random() * 100) + 1` (create a random number between 1 and 100)

Reply:
Sure! I agree this will be more attractive, and I think it is alright if there is a 1/100 chance the output does not match listed output ("Modified 1 Document") on runs after the first run.